samples/Sandbox/TagHelpers/NoopTagHelper.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Sandbox.TagHelpers
+{
+    [HtmlTargetElement(Attributes = "active")]
+    public class NoopTagHelper : TagHelper
```

Comment:
It's all for tooling. Maybe `sample` is a bad word for this commit. It's just to validate what you've done works.